builders/interoperability/xcm/xc20/send-xc20s/overview.md

```diff
@@ -1,46 +1,48 @@
+For reference, the XCM extrinsics used by the Polkadot XCM Pallet are defined at the top of the [Using the Polkadot XCM Pallet To Send XC-20s guide](/builders/interoperability/xcm/xc20/send-xc20s/xcm-pallet/){target=_blank}.
 
-The set of XCM instructions to be used depends on the token being transferred and the route taken. For example, there is one set of XCM instructions for sending the native asset back to its origin chain (reserve chain), such as xcDOT from Moonbeam back to Polkadot, and another set of XCM instructions for sending the native asset from the origin chain to a destination chain, such as DOT from Polkadot to Moonbeam.
+The specific instructions in each XCM depend on the asset being transferred and the route taken. For example, sending a native asset back to its reserve chain (for instance, xcDOT from Moonbeam back to Polkadot) differs from sending it from its origin to a new chain (DOT from Polkadot to Moonbeam).
 
-The following sections provide some examples of the XCM instructions involved in token transfers via XCM.
+Examples of the XCM instructions typically involved in token transfers are included below.
 
 ### Instructions to Transfer a Reserve Asset from the Reserve Chain {: #transfer-native-from-origin }
 
 --8<-- 'text/builders/interoperability/xcm/xc20/send-xc20s/overview/DOT-to-xcDOT-instructions.md'
 
-For more information on constructing an XCM message to transfer self-reserve (native) assets to a target chain, such as DOT to Moonbeam, you can refer to the guide to the [Polkadot XCM Pallet](/builders/interoperability/xcm/xc20/send-xc20s/xcm-pallet/){target=\_blank}.
+For more information on constructing an XCM message for self-reserve (native) assets—such as DOT to Moonbeam—refer to the [Polkadot XCM Pallet guide](https://docs.moonbeam.network/builders/interoperability/xcm/xc20/send-xc20s/xcm-pallet/){target=_blank}.
 
-It calls `TransferReserveAsset` and passes in `assets`, `dest`, and `xcm` as parameters. In particular, the `xcm` parameter includes the `BuyExecution` and `DepositAsset` instructions. If you then head over to the Polkadot GitHub repository, you can find the [`TransferReserveAsset` instruction](https://github.com/paritytech/polkadot-sdk/blob/{{ polkadot_sdk }}/polkadot/xcm/xcm-executor/src/lib.rs#L671){target=\_blank}. The XCM message is constructed by combining the `ReserveAssetDeposited` and `ClearOrigin` instructions with the `xcm` parameter, which, as mentioned, includes the `BuyExecution` and `DepositAsset` instructions.
+This process calls `TransferReserveAsset` with `assets`, `dest`, and `xcm` parameters. Within `xcm`, the typical instructions are `BuyExecution` and `DepositAsset`. The [`TransferReserveAsset` instruction](https://github.com/paritytech/polkadot-sdk/blob/{{ polkadot_sdk }}/polkadot/xcm/xcm-executor/src/lib.rs#L671){target=_blank} on GitHub includes `ReserveAssetDeposited` and `ClearOrigin` alongside `BuyExecution` and `DepositAsset` to finalize the transfer.
```

Comment:
How can they track back where the `xcm` parameter is assembled? The whole point of having these links here are so that devs can make sense of it and see for themselves how the messages are built.

The link, which points to line 671, is referencing a line in the `ReceiveTeleportedAsset` function. So that needs to be updated. It should be 646, I believe. But that alone is confusing. There used to be a paragraph above this one that referenced calling the `TransferReserveAsset` function, including the assembly of the `xcm` instructions, but it's no longer here because it referenced X-Tokens, but there should be an equivalent elsewhere now.

I feel like as it is now, yes, it does make sense, but we're not connecting all of the dots.